src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -187,7 +187,7 @@ public void produceHourlySnapshotsForOrg(String orgId) {
 
           tallyStateRepository.update(currentState);
           recordTallyCount(totalSnapshots.values().stream().flatMap(Collection::stream).toList());
-          summaryProducer.produceTallySummaryMessages(totalSnapshots);
+          summaryProducer.produceTallySummaryMessages(totalSnapshots, List.of(Granularity.HOURLY));
```

Comment:
@mstead can you confirm that the hourly operation won't produce daily snapshots as part of this process? 
We have two cronjobs, one for hourly and another one for daily, but I do recall that regardless to when the process is triggered, it could produce snapshots using either hourly, daily, monthly, ... granularities.